docs/source/posts/2023/2023-06-27-week-4-tvcastillod.rst

```diff
@@ -0,0 +1,30 @@
+Week 4: First draft of the DTI uncertainty visualization
+========================================================
+
+.. post:: June 27, 2023
+   :author: Tania Castillo
+   :tags: google
+   :category: gsoc
+
+What did I do this week?
+------------------------
+
+`#PR 810: DTI uncertainty visualization <https://github.com/fury-gl/fury/pull/810>`_
+
+I made a second PR with the implementation of DTI uncertainty calculation and visualization. Below is an image of diffusion tensor ellipsoids and their associated uncertainty cones.
+
+.. image:: https://user-images.githubusercontent.com/31288525/254747296-09a8674e-bfc0-4b3f-820f-8a1b1ad8c5c9.png
+    :width: 530
+    :align: center
+
+I had to use some **dipy** functions, specifically: `estimate_sigma <https://dipy.org/documentation/1.4.1./reference/dipy.denoise/#estimate-sigma>`_ for the noise variance calculation, `design_matrix <https://dipy.org/documentation/1.4.0./reference/dipy.reconst/#design-matrix>`_ to get the b-matrix, and `tensor_prediction <https://dipy.org/documentation/1.4.0./reference/dipy.reconst/#tensor-prediction>`_ for the signal estimation. The details of this calculations can be found `here <https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.21111>`_.
```

Comment:
can you replace all link by a permalink on DIPY github repo? 

the version 1.4.1 of DIPY is old, links can changed